conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -12,9 +12,9 @@
 # Add all the application routes to the prod.routes file
 
 ### NoDocs ###
-GET   /docs/swagger-ui/*file                controllers.Assets.at(path:String="/public/lib/swagger-ui", file:String)
+GET   /test-only/docs/swagger-ui/*file                controllers.Assets.at(path:String="/public/lib/swagger-ui", file:String)
```

Comment:
are these /test-only routes being used anywhere? do we really need to make these changes?

Reply:
I am not sure where it has been used should, I remove /test-only here.